### PR TITLE
Adding the SIGTERM to soft exit

### PR DESCRIPTION
**Pull request by arthurguedes375**
When I was using  docker-compose I could not generate the flame-graph because the default signal sent by docker-compose is SIGTERM, there are ways to send SIGINT instead of SIGTERM but that adds more complexity to the docker-compose file, and since SIGTERM is not a force exit signal I think that it should be handled the same way that SIGINT is.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "0x",
-  "version": "5.0.0",
+  "version": "5.0.1",
```
Comment (mcollina) [inline package.json:3]: Can you revert this? We'll bump it on releaze

**Review by mcollina — COMMENTED** [review]
Good work! Could you document this in the README?

File: package.json (hunk 1 above)
Comment (arthurguedes375) [reply package.json]: Sure, i got it!

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Pull request merged**